src/catch2/internal/catch_getenv.cpp

```diff
@@ -14,7 +14,7 @@
 namespace Catch {
     namespace Detail {
 
-#if defined( CATCH_PLATFORM_WINDOWS_UWP )
+#if ! defined (CATCH_CONFIG_GETENV)
```

Comment:
```suggestion
#if !defined (CATCH_CONFIG_GETENV)
```

just a stylistic nit

Reply:
I can't add a note to unchanged line, so I am adding this one here: you are missing include of `catch_compiler_capabilities` in this file, so the tests break (because the tests no longer can see env variables).